CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+- `hostmetricsreceiver`: Add cpu.utilization metrics to cpu scrapper (#6221)
```

Comment:
I should've clarified that this should be still part of Enhancements but Unreleased section, not a released version :)

Reply:
I've thought about it too but I :) I'll change it 👍